### PR TITLE
Refactor Containers related methods & variables

### PR DESCRIPTION
**Pull request by rgayon**
Also make sure functions use Container objects as arguments and not container_id.

### CODE REVIEW

File: docker_explorer/lib/storage.py [hunk 1]
```diff
@@ -66,21 +66,21 @@ def GetAllContainersInfo(self):
             'If it is correct, you might want to run this script'
             ' with higher privileges.').format(
                 self.container_config_filename, self.docker_directory)
-    container_info_list = [self.GetContainerInfo(x) for x in container_ids_list]
+    containers_list = [self.GetContainer(x) for x in container_ids_list]
```
Comment (tomchop) [inline docker_explorer/lib/storage.py]: What about just returning the list comprehension?

File: docker_explorer/lib/storage.py [hunk 2]
```diff
@@ -233,18 +233,18 @@ def GetLayerInfo(self, container_id):
...
+        return layer_info
     return None
 
-  def _MakeExtraVolumeCommands(self, container_info, mount_dir):
+  def _MakeExtraVolumeCommands(self, container_obj, mount_dir):
     """Generates the shell command to mount external Volumes if present.
 
     Args:
-      container_info (dict): the container's metadata.
+      container_obj (Container): the container object.
```
Comment (tomchop) [inline docker_explorer/lib/storage.py]: Remove space before type

File: docker_explorer/lib/storage.py [hunk 3]
```diff
@@ -301,18 +301,18 @@ def Mount(self, container_id, mount_dir):
...
         subprocess.call(c, shell=True)
 
-  def GetHistory(self, container_id, show_empty_layers=False):
+  def GetHistory(self, container_obj, show_empty_layers=False):
     """Returns a string representing the modification history of a container.
 
     Args:
-      container_id (str): the ID of the container.
+      container_obj(Container): the container object.
       show_empty_layers (bool): whether to display empty layers.
```
Comment (tomchop) [inline docker_explorer/lib/storage.py]: Remove space before type

File: docker_explorer/lib/storage.py (hunk 1 above)
Comment (rgayon) [reply docker_explorer/lib/storage.py]: done

File: docker_explorer/lib/storage.py (hunk 3 above)
Comment (rgayon) [reply docker_explorer/lib/storage.py]: added =)

File: docker_explorer/lib/storage.py (hunk 2 above)
Comment (rgayon) [reply docker_explorer/lib/storage.py]: no!

**Pull request merged**